controllers/mobile/frmAgendaController/frmAgendaController.js

```diff
@@ -846,6 +846,7 @@ define({
      * 	@private
      */
     setSpeakerProfile: function(eventObject) {
+      	this.view.flxRatingContainer.isVisible=true;
```

Comment:
Validate the below flow.

1.Navigate to a particular Agenda Detail
2.Submit the Feedback
3.Navigate to Agenda List
4.Open Agenda Detail of some other agenda

It is directly scrolling to the position where the feedback is present

Reply:
handled animation